src/components/Filters/Filters.tsx

```diff
@@ -211,6 +218,11 @@ class Filters extends React.Component<ComposedProps, State> {
       </div>
     );
 
+    const filterResourceName = resourceName || {
```

Comment:
resourceName can be undefined. Something to note for the future, maybe we should just make the english translation the default?